lib/internal/freelist.js

```diff
@@ -9,9 +11,15 @@ class FreeList {
   }
 
   alloc() {
-    return this.list.length ?
-      this.list.pop() :
-      this.ctor.apply(this, arguments);
+    let item;
+    if (this.list.length) {
```

Comment:
Can you make this check explicit? Turbofan prefers it to implicit boolean conversions.

Reply:
Done. Whatever makes Turbofan happy :-)